Libraries/Components/TextInput/TextInput.js

```diff
@@ -1601,6 +1602,13 @@ const ExportedForwardRef: React.AbstractComponent<
     React.ElementRef<HostComponent<mixed>> & ImperativeMethods,
   >,
 ) {
+  let style = flattenStyle(restProps.style);
```

Comment:
Since we are not assigning this value let use const here 
```suggestion
  const style = flattenStyle(restProps.style);
```

Reply:
Sure, fixed. Thanks for your review!